src/styles/catalog.scss

```
@@ -0,0 +1,21 @@
+@import "general";
+
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, $card-width);
+  justify-content: center;
+  row-gap: $card-gap;
+  column-gap: 48px;
```

```suggestion
  gap: $card-gap 48px;
```
І винеси в змінну 48px;